### PR TITLE
[WI00664559] APInvoice docs update adding cluster and lineItems

### PR DESCRIPTION
**Pull request by jfvanin**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -474,6 +474,9 @@ The following objects can be used as parameters in the *include* query
 | files/packingList                       | Details of the group's Packing Lists                               |
 | files/packingList/lineItem              | List of line items associated with the Packing List                |
 | files/apInvoice                         | Details of the Accounts Payable Invoice                            |
+| files/apInvoice/cluster                 | List of clusters associated with Payable Invoice                   |
+| files/apInvoice/cluster/jobReference    | List of References associated with Payable Invoice's cluster       |
+| files/apInvoice/cluster/extractedLine   | List of extracterd Line associated with Payable Invoice'cluster    |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:479]: There is a spelling mistake in Invoice's cluster ..
And also may be it is better to say "List of extracted charge lines associated with Payable invoice's cluster"

File: source/index.html.md [hunk 2]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
 | files.apInvoice.departmentCode            |  The department code of this invoice.             |
 | files.apInvoice.branchCountry            |   The branch country of this invoice.             |
 
+### *Files/apInvoice/Cluster* attributes
+
+| Attribute                               |  Description                                                                                                                      |
+| --------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| files.apInvoice.cluster.total   | The cluster total Value                                                      |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1025]: Each sub-total section on the invoice is considered a cluster, So cluster total basically is a sub-total

File: source/index.html.md [hunk 3]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.description | The cluster description                                                  |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1026]: Textual description of a subtotal on the invoice

File: source/index.html.md [hunk 4]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.glCode         | The Cluster GL Code                                                      |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1027]: A general ledger code

File: source/index.html.md [hunk 5]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.vatTotal         | The Cluster total VAT Value                                                  |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1028]: Total tax amount of the cluster

File: source/index.html.md [hunk 6]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.taxCode         | Tax Code                                                     |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1029]: Not extracted

File: source/index.html.md [hunk 7]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.branch         | The Branch code                                                      |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1030]: Not extracted

File: source/index.html.md [hunk 8]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.departmentCode         | The Department Code                                                      |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1031]: Not extracted

File: source/index.html.md [hunk 9]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.orderIndex         | The position which the cluster is displayed in relation with the other clusters                     |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1032]: Not extracted

File: source/index.html.md [hunk 10]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.jobReference.jobRef         | The Job Ref                                                      |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1033]: The shipment or consol reference

File: source/index.html.md [hunk 11]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.jobReference.bolNum         | The Bol Number                                                      |
+| files.apInvoice.cluster.jobReference.containerNum         | The Container  Number                                                      |
+| files.apInvoice.cluster.jobReference.purchaseOrder         | The Purchase Order                                                     |
+| files.apInvoice.cluster.jobReference.serviceStartDate         |  The Service start date                                                    |
+| files.apInvoice.cluster.jobReference.serviceEndDate         | The Service end date                                                      |
+| files.apInvoice.cluster.jobReference.orderIndex         | The position which the Job ref is displayed in relation with the other refs         |
+| files.apInvoice.cluster.extractedLine.service         | The Service of the extracted Line       |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1040]: The service of the charge line

File: source/index.html.md [hunk 12]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.extractedLine.journey         | The Journey of the extraxted Line       |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1041]: The journey of the charge line

File: source/index.html.md [hunk 13]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.extractedLine.unitPrice         | The Unit price of the extracted Line         |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1042]: The unit price of the charge line

File: source/index.html.md [hunk 14]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.extractedLine.quantity         | The quantity of the extracted Line         |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1043]: The quantity of the charge line

File: source/index.html.md [hunk 15]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.extractedLine.currency         | The currency of the extraxcted Line        |
+| files.apInvoice.cluster.extractedLine.lineVat         | The VAT of the extracted Line         |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1045]: Total tax amount of the charge line

File: source/index.html.md [hunk 16]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.extractedLine.lineGross         | The Gross value of the extracted Line     |
+| files.apInvoice.cluster.extractedLine.exchangeRate         | The exchange rate of the currency of the extracted line       |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1047]: The exchange rate of the charge line

File: source/index.html.md [hunk 17]
```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
...
+| files.apInvoice.cluster.jobReference.serviceStartDate         |  The Service start date                                                    |
+| files.apInvoice.cluster.jobReference.serviceEndDate         | The Service end date                                                      |
+| files.apInvoice.cluster.jobReference.orderIndex         | The position which the Job ref is displayed in relation with the other refs         |
+| files.apInvoice.cluster.extractedLine.service         | The Service of the extracted Line       |
+| files.apInvoice.cluster.extractedLine.journey         | The Journey of the extraxted Line       |
+| files.apInvoice.cluster.extractedLine.unitPrice         | The Unit price of the extracted Line         |
+| files.apInvoice.cluster.extractedLine.quantity         | The quantity of the extracted Line         |
+| files.apInvoice.cluster.extractedLine.currency         | The currency of the extraxcted Line        |
+| files.apInvoice.cluster.extractedLine.lineVat         | The VAT of the extracted Line         |
+| files.apInvoice.cluster.extractedLine.lineGross         | The Gross value of the extracted Line     |
```
Comment (SowmyaLakkegowda29) [inline source/index.html.md:1046]: The gross value of the charge line

**Review by SowmyaLakkegowda29 — CHANGES_REQUESTED** [review]
Needs changing some descriptions.

Some attributes are not extracted so It is better to remove them here and in API

**Review by jfvanin — COMMENTED** [review]
Thanks for the review Sowmya, I think I've covered it all

**Pull request merged**